crates/graphql/src/schema/dataloaders/nft.rs

```diff
@@ -168,3 +168,13 @@ impl TryBatchFn<PublicKey<Nft>, Vec<NftActivity>> for Batcher {
             .batch(addresses))
     }
 }
+
+#[async_trait]
+impl TryBatchFn<PublicKey<NftActivity>, Nft> for Batcher {
+    async fn load(
+        &mut self,
+        addresses: &[PublicKey<NftActivity>],
+    ) -> TryBatchMap<PublicKey<NftActivity>, Nft> {
+        let conn = self.db()?;
+    }
+}
```

Comment:
I needed this very loader type for feed_events. Can you take this one and add here.

https://github.com/holaplex/indexer/pull/368/files#diff-beaf2efe58be82e65ef92d01fba2f303960ebe4530074a71288cbf4eb78d34e3R173